### ISSUE TITLE
Request: remove the need of va_ext.h (from proprietary Media Server Studio) for building

### ISSUE THREAD

**Issue by dbermond** [Intel-Media-SDK/MediaSDK#28]
Currently, MediaSDK needs `CL/va_ext.h` in order to build (details [here](https://github.com/Intel-Media-SDK/MediaSDK/blob/e2b2f3d41a54cf60aeba881c7684359f61d7b8d0/samples/sample_plugins/rotate_opencl/include/opencl_filter_va.h#L33)).

As far as I could gather, this header file is provided only by the proprietary Media Server Studio (MSS). It means that MSS is still needed for building.

Since MediaSDK is opensource, it would be good if there was a way to completely remove the necessity of dealing with the proprietary MSS, specially now that proprietary libva from MSS is not needed any more. Also, MSS is a huge download for just a single header file.

Can this be implemented? Thanks.

**Comment by ruiling** [comment]
@dbermond Could you give it a try: #21 ? to have a check whether the commit "Detect Intel OpenCL SDK exactly." solve your issue or not.

**Comment by dbermond** [comment]
@ruiling Thank you for the reply. The opencl patch from #21 does not solve this for me.

After uninstalling the files from MSS and applying the patch, I'm getting:

```
-- Intel OpenCL SDK not found
-- OpenCL was found here: /usr/lib and /usr/include
```

```
samples/sample_plugins/rotate_opencl/include/opencl_filter_va.h:33:23: fatal error: fatal error: CL/va_ext.h: No such file or directory
 #include <CL/va_ext.h>
                       ^
compilation terminated.
```

**Comment by ruiling** [comment]
@dbermond I slightly update the #21, I think it can fix your issue. I may made some mistake last time I was building the project. In fact some extra cmake commands should also be put under INTEL_OPENCL_SDK. Would you please give it a try? thanks!

**Comment by dbermond** [comment]
@ruiling Thank you for modifying the pull request. I can confirm that the updated commit works for me. It removes the need of `CL/va_ext.h` from MSS.

**Comment by dbermond** [comment]
This was implemented by commit cb58f7d7e9994ef6167ecadef9feaf023cf6481b. Thank you.

Closing.
